react/features/landing/reducer.js

```diff
@@ -0,0 +1,21 @@
+import { ReducerRegistry } from '../base/redux';
+
+import { LANDING_IS_SHOWN } from './actionTypes';
+
+ReducerRegistry.register('features/landing', (state = {}, action) => {
+    switch (action.type) {
+    case LANDING_IS_SHOWN:
+        return {
+            ...state,
+
+            /**
+             * Flag that shows that mobile landing shown shown.
+             *
+             * @type {App}
```

Comment:
The type of the property `landingIsShown` is definitely not `App`, it is `boolean`.